packaging/greenboot/bootc-rollback-os.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -x
+set -eo pipefail
+
+LOGFILE="/var/log/rollback.log"
+TIMESTAMP=$(date "+%Y-%m-%d %H:%M:%S")
+
+rollback() {
```

Comment:
When a script in `/usr/lib/greenboot/check/required.d/` fails, greenboot should _automatically_ reboot twice into the new version, and on the 3rd attempt roll-back.